kotlin/src/main/kotlin/necklace/juwellery/Juwellery.kt

```diff
@@ -0,0 +1,40 @@
+package necklace.juwellery
+
+import necklace.enums.EarringType
+import necklace.enums.Juwel
+import necklace.enums.NecklaseType
+
+
+sealed class Juwellery(open val stone: Juwel) {
+    open val isSmall = false
```

Comment:
To be honest, I don't understand these two properties. What would be the meaning of a juwellery with `isSmall == true` and `isLarge == true`? I assume `isSmall == false` and `isLarge == false` means it's "medium size".

How about using a "size class" enum with values "Small", probably "Medium", and "Large"?

Reply:
yes that would actually make more sense with an enum of 3 values